membership-attribute-service/app/prodtest/Allocator.scala

```diff
@@ -0,0 +1,14 @@
+package prodtest
+
+class Allocator {
+
+  def isInTest(identityId: String, percentageInTest: Double): Boolean = {
+    if(percentageInTest > 0) {
+      val cleaned = identityId.replaceFirst("^0+", "").toInt
+      val index = cleaned % 100
+
+      index <= percentageInTest
```

Comment:
...however this should be < rather than <= (as `index` will be between 0-99 includive)